src/question-two/QuestionTwo.tsx

```diff
@@ -4,13 +4,17 @@ import { IAppTabContainer } from "../common/types"
 import { SectionGroup } from "../components/section/SectionGroup"
 import { SectionPanel } from "../components/section/SectionPanel"
 
-import "./QuestionTwo.css"
-
-/**
- * Please do not change these dates, the data on the server all fall within the 01/09/2018
- */
-const RANGE_START = new Date("2018-09-01T00:00:00Z")
-const RANGE_END = new Date("2018-09-01T24:00:00Z")
+interface Lane {
+  title: string
+  cards: {
+    allocId: number,
+    allocType: 'job' | 'activity',
+    resourceId: number,
+    description: string,
+    start: string,
+    end: string
+  }[]
+}
```

Comment:
Should we still call this a `Lane` with cards inside? At first I thought it could probably be a bit more ambiguous, but in the end I guess it doesn't matter _what_ we call it. So long as the shape and data matches.

Reply:
`Lane` doesn't really mean anything now. We are wanting people to merge the schedule data of a resource, so how 'bout `ResourceSchedule`? Likewise, some of the keys are off. Maybe instead of `title` it's called `name` and `resourceId` sits outside of cards?